### PR TITLE
Splash screen 1105

### PR DESCRIPTION
**Pull request by teresam856**
Rough PR... for [Issue 1105](https://github.com/GMOD/jbrowse-components/issues/1105)
Current State

- Redirect when no default Views

HmRedirect

- Default Config/View

defaultViewVolvox

- TODO: 
- [x] Add SplashScreen
- [x] Add Session list/thumbnails
- [x] Add Session Deleting capability
- [ ]  ~Add Session Renaming capability~
- [x] Sort out redirects
- [x] Button to go back to splash Screen
- [x] Testing

### CODE REVIEW

File: packages/core/ui/NewSessionCards.js [hunk 1]
```diff
@@ -93,10 +99,48 @@ ProceedEmptySession.propTypes = {
...
+
+    // root.addSavedSession({ name: snapshot.name })
+
+    localStorage.setItem(
+      `autosave-${root.configPath}`,
+      JSON.stringify({
+        session: {
+          ...snapshot,
+          name: `${snapshot.name}-autosaved`,
+        },
```
Comment (cmdcolin) [inline packages/core/ui/NewSessionCards.js:134]: there are autoruns on the rootmodel that will do this localstorage setItem automatically for both autosave- and current

Comment (teresam856) [reply packages/core/ui/NewSessionCards.js:145]: Ahhh noted, will fix that.

**Comment by teresam856** [comment]
In terms of sessions, still need to add pagination and ability to rename the sessions. Need a button to redirect back to the splashScreen.

**Comment by cmdcolin** [comment]
I don't think we need pagination. If needed you can set a height on a div containing sessions and overflow: auto

```
<div style={{overflow:'auto', height:400}}>
session list...
</div>
```

**Comment by cmdcolin** [comment]
+1 for return to splash screen though if possible

**Comment by teresam856** [comment]
@cmdcolin Ok, I am wrapping up the renaming a session part of the recent session card, and I will get working on the redirect back to splashScreen. I pushed my most recent changes in case you want to take a look at the current state without the session renaming functionality

**Comment by cmdcolin** [comment]
went ahead and pushed a "return to splash screen" menu item direct to this branch :)

**Comment by teresam856** [comment]
@cmdcolin thanks Colin!
I am about to push the rename functionality. Just fixing a bug related to updated the cards after changing the name of the session.

**Comment by cmdcolin** [comment]
nice.... I did a couple other random things on the branch, let me know if there's anything looks weird...

**Comment by cmdcolin** [comment]
hopefully doesnt cause merge conflict with your local work but happy to help if so...

**Comment by teresam856** [comment]
Leaving out renaming session functionality from splash screen out from JBrowse web. The reasoning behind this
* more reasonable for users to rename a session that they are currently using
* the delete functionality is still within the splash screen for easy delete 
* avoids session saving/ session naming errors if updates do not trigger appropriate re-renders

**Comment by teresam856** [comment]
backToSplashScreen
splashScreen
deleteSessionDialog
deleteIcon

**Comment by teresam856** [comment]
Screen Shot 2020-11-04 at 11 50 42 AM

**Review by rbuels — APPROVED** [review]
code looks good! I'd be fine with it being merged

**Comment by teresam856** [comment]
Removed thumbnails that are set by default, and made sessionNames more visible
Screen Shot 2020-11-04 at 1 54 57 PM

**Review by peterkxie — APPROVED** [review]
Looks good! asked you some stuff about analytics over gitter to make sure they are still firing at the appropriate time

**Review by garrettjstevens — APPROVED** [review]

**Pull request merged**